src/uwtools/tests/config/test_jinja2.py

```diff
@@ -217,6 +257,20 @@ def test_render_values_needed(caplog, values_file, template):
         assert logged(caplog, var)
 
 
+def test__dry_run_template(caplog):
+    jinja2._dry_run_template("roses are red, violets are blue")
+    assert logged(caplog, "roses are red, violets are blue")
```

Comment:
It would be stronger if the argument were two lines, to exercise the fact that the function-under-test splits on newlines and loops on those. So e.g.
```
jinja2._dry_run_template("roses are red\nviolets are blue")
assert logged(caplog, "roses are red")
assert logged(caplog, "violets are blue")
```
since those two lines would be logged separately.